apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/BorrowDetailsGraph.tsx

```diff
@@ -8,17 +8,30 @@ import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRen
 import { StatisticsCard } from '../../../../2_molecules/StatisticsCard/StatisticsCard';
 import { useIsMobile } from '../../../../../hooks/useIsMobile';
 import { translations } from '../../../../../locales/i18n';
+import { Chart } from './components/Chart/Chart';
+import { harcodedData } from './components/Chart/Chart.constants';
+import { MockData } from './components/Chart/Chart.types';
 
 const pageTranslations = translations.aaveReserveOverviewPage.borrowDetails;
 
 type BorrowDetailsGraphProps = {};
 
-// TODO: mocked amounts
-
 export const BorrowDetailsGraph: FC<BorrowDetailsGraphProps> = () => {
   const [open, setOpen] = useState<boolean>(true);
   const { isMobile } = useIsMobile();
 
+  // TODO: mocked amounts
+  const mockData: MockData<{ x: string; y: number }> = useMemo(() => {
+    const data1 = harcodedData;
+
+    return {
+      data1,
+      label1: 'Borrow APR, variable',
+      lineColor: '#72eadf',
```

Comment:
- Unless the label will be a mock, I think the best place is to be with the translations.
- lineColor could be in the constants file.